skin_generator/generator.cpp

```diff
@@ -96,7 +96,7 @@ void SkinGenerator::ProcessSymbols(std::string const & svgDataDir,
           QSize defaultSize = m_svgRenderer.defaultSize();
 
           QSize symbolSize = symbolSizes[j];
-          QSize size = defaultSize * (symbolSize.width() / 24.0);
+          QSize size = defaultSize * (symbolSize.width() / 18.0);
```

Comment:
Can we clarify here where the 18 constant comes from?
Is it something that m_svgRenderer.defaultSize is 18?
Should we still keep this transformation when we adjusted all the svg dimensions (in later commits)?

Reply:
I don't understand it too and it'll be great to clarify this logic.

Not necessary in this PR.

(I wanted to go through the code and learn it myself, but don't have time now)

Reply:
As far as I know this is how this code works:
* `m_svgRenderer.defaultSize()` is the size of the SVG file (After the resize in this PR generally the -m files are 18x18 and the -s files are 12x12 ─ previously -m files were 24x24 and -s files were 16x16)
* `symbolSize` is the size of the PNG skins that we want to generate and they are defined in the script [generate_symbols.sh](https://github.com/organicmaps/organicmaps/blob/master/tools/unix/generate_symbols.sh#L82-L94): 18 (mdpi), 27 (hdpi), 36 (xhdpi)... 
* The `18.0` comes from the fact that the icons that should be represented as 18x18 in mdpi are also saved as 18x18 SVG files. Previously the constant `24.0` came from the fact that the icons that should be represented as 18x18 in mdpi were saved as 24x24 SVG files.

Reply:
Many thanks! Could you please convert this knowledge into the code comments?

Reply:
Added comments